src/components/Articles.jsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from "react";
+import { allArticles } from "../api";
+import ArticleCard from "./ArticleCard";
+
+export default function Articles() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    allArticles.then((articles) => {
+      setArticles(articles.data.articles);
```

Comment:
This is confusingly named! I'd suggest either de-structuring data on line 9, or calling the response something more generic (e.g response). Only the final .articles truly represents an array of articles.

Reply:
Please include a loading state to feed back to users while they are waiting for the page to render.